### PR TITLE
Rename RPC endpoints + update main nav order

### PR DESCRIPTION
**Pull request by quietbits**
- Rename main nav "Endpoints" to "API Explorer"
- Rename "RPC Endpoints" to "RPC Methods" (keep "Horizon Endpoints" to match the docs)
- Put "Accounts" and "Transactions" next to each other in the main nav

### CODE REVIEW

File: src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx [hunk 1]
```diff
@@ -198,7 +198,7 @@ export const SavedEndpointsPage = () => {
         heading={{ title: "Saved Endpoints" }}
```
Comment (quietbits) [inline src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx:198]: I'm not sure how to name "Saved Endpoints". :thinking: "Saved Endpoints and Methods" seems too long. Any ideas, @jeesunikim @janewang @sdfcharles @leighmcculloch ? :pray:

Comment (janewang) [reply src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx:198]: @quietbits How about `Saved API Requests`?

Comment (leighmcculloch) [reply src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx:198]: I like @janewang's suggestion. I'd shorten it to: Saved Requests.

Comment (quietbits) [reply src/app/(sidebar)/endpoints/components/SavedEndpointsPage.tsx:198]: Thank you both! :raised_hands: I'll go with "Saved Requests".

**Pull request merged**